app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,89 @@
+      android:layout_marginEnd="20dp">
+
+      <ImageView
+        android:id="@+id/btn_remove"
+        android:layout_width="24dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:layout_alignParentEnd="true"
+        android:contentDescription="@string/remove"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_clear_24dp" />
+
+      <Spinner
         android:id="@+id/spinner_description_languages"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_weight="0"
-        android:minWidth="@dimen/tiny_height"
-        android:padding="@dimen/dimen_0"
+        android:layout_alignParentStart="true"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         android:spinnerMode="dialog" />
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:layout_weight="8">
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/caption_item_edit_text_input_layout"
+      android:id="@+id/caption_item_edit_text_input_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/caption_item_edit_text"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        >
-
-        <androidx.appcompat.widget.AppCompatEditText
-            android:id="@+id/caption_item_edit_text"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:hint="@string/share_caption_hint"
-            android:imeOptions="actionNext|flagNoExtractUi"
-            android:inputType="textMultiLine" />
+        android:layout_height="match_parent"
+        android:hint="@string/share_caption_hint"
+        android:imeOptions="actionNext|flagNoExtractUi"
+        android:inputType="textMultiLine" />
     </com.google.android.material.textfield.TextInputLayout>
 
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/description_item_edit_text_input_layout"
+      android:id="@+id/description_item_edit_text_input_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
```

Comment:
if you have the same margin on start/end there is the attribute `android:layout_marginHorizontal` and of course a `vertical` variant should top/bottom be the same